### DISCUSSION TITLE
How do I Use logs in go-imgui

### DISCUSSION THREAD

**Discussion by sohailjs** [AllenDang/giu#816] [Q&A] [answered]
There is log example in demo example. I wanted the same for go. How can i do that?

**Comment by gucio321** [comment]
I think this is `ImGui::ShowDebugLogWindow`
and it is `func ShowDebugLogWindowV(p_open *bool) {` in cimgui-go

**Reply by sohailjs** [reply]
i want to create that type of logging for my project. How can i do that?

**Reply by gucio321** [reply]
well, I'd use InputTextMultiline with ReadOnly flag, and set it as output of `log` package.

**Reply by sohailjs** [reply]
so i used InputTextMultiline  with autoScrollToBottom on. But now when i scroll up manually it doesn't stay there and keeps taking me to bottom because of autoScroll is on. Is there any way to handle this?

**Reply by AllenDang** [reply, marked as answer]
@sohailjs you could use an variable to control whether to active autoScroll, in your case, you could detect manual scroll and set auto scroll to false.

**Reply by sohailjs** [reply]
Got it. Thanks
